apps/web-giddh/src/assets/locale/all-features/mr.json

```diff
@@ -151,5 +151,24 @@
 	"support": "आधार",
 	"talk_to_sales": "विक्रीशी बोला",
 	"help_doc": "मदत दस्तऐवज",
-	"schedule_demo": "डेमोचे वेळापत्रक तयार करा"
+	"schedule_demo": "डेमोचे वेळापत्रक तयार करा",
+    "purchase_bill": "Purchase Bill",
```

Comment:
Please add marathi translation here

Reply:
Not provided by sandhya, Will update once I get